pharus/component_interface.py

```diff
@@ -75,6 +106,13 @@ def restriction(self):
                             r"^datetime.*$",
                             self.fetch_metadata["query"].heading.attributes[k].type,
                         )
+                        else datetime.fromtimestamp(float(v))
```

Comment:
Have we tried some form of `isoformat()` to see if it works in a general way for both `date` and `datetime`?

For example: 

```python
>>> datetime.fromtimestamp(float(v)).isoformat()
'2022-03-18T10:48:00.123000'
```